File: JAVASCRIPT/Questões uri/1015.js
let pegarValores = (line) => line.split(" ").map(a => +a)
let [x1,y1] = pegarValores(lines.shift())

let pegarValoress = (line) => line.split(" ").map(a => +a)
let [x2,y2] = pegarValoress(lines.shift())

let soma= Math.sqrt(((x2-x1)*(x2-x1))+((y2-y1)*(y2-y1)))

console.log(soma.toFixed(4))

File: ESTUDO DE LINGUAGENS/PYTHON/Questões Uri/1015.py
import math

lenha=input().split(" ")
x1, y1= lenha

linha=input().split(" ")
x2, y2= linha

dist= math.sqrt((((float(x2) - float(x1))*(float(x2) - float(x1))) + ((float(y2)-float(y1)) *(float(y2)-float(y1)))))
print("%0.4f"%dist)